### ISSUE TITLE
What did 3.1.2 do with polydispersity in the Fractal Model; what should 4.2 do? (Trac #877)

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#963]
Greg Newbloom & Adrian Rennie have both commented on the removal of polydispersity on the radius parameter in the fractal model in 4.0.1. It was allowed in earlier versions.

The fractal S(Q) in this model ''also'' includes the radius parameter (as well as the spherical P(Q)).

So there are two questions:

1. Did 3.1.2 and earlier versions only apply polydispersity to P(Q), or to P(Q) ''and'' S(Q)?

2. Should polydispersity be put back into this model for 4.2?

Migrated from http://trac.sasview.org/ticket/877
```json
{
    "status": "closed",
    "changetime": "2017-03-20T03:23:43",
    "_ts": "2017-03-20 03:23:43.523229+00:00",
    "description": "Greg Newbloom & Adrian Rennie have both commented on the removal of polydispersity on the radius parameter in the fractal model in 4.0.1. It was allowed in earlier versions.\n\nThe fractal S(Q) in this model ''also'' includes the radius parameter (as well as the spherical P(Q)).\n\nSo there are two questions:\n\n1. Did 3.1.2 and earlier versions only apply polydispersity to P(Q), or to P(Q) ''and'' S(Q)?\n\n2. Should polydispersity be put back into this model for 4.2?\n",
    "reporter": "smk78",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Bug Fixing",
    "time": "2017-03-09T10:15:53",
    "component": "SasView",
    "summary": "What did 3.1.2 do with polydispersity in the Fractal Model; what should 4.2 do?",
    "priority": "critical",
    "keywords": "",
    "milestone": "SasView 4.1.0",
    "owner": "butler",
    "type": "task"
}
```

**Comment by butlerpd** [comment]
Trac update at `2017/03/09 14:39:17`: 
* **butler** commented:

Clearly my fault. The git record clearly shows that I converted this model and failed to make radius a volume parameter, while Piotr converted the fractal core-shell and DID make radius and thickness polydisperse.  Clearly this was done as an oversight not deliberately due to some deep problem.  I will correct this for 4.1 


* **butler** changed milestone from "SasView 4.2.0" to "SasView 4.1.0"
* **butler** changed owner from "" to "butler"
* **butler** changed status from "new" to "assigned"

**Comment by butlerpd** [comment]
Trac update at `2017/03/20 03:23:43`: 
* **butler** commented:

In changeset 478882278efdd578c67d41a151c1209805308f3f:

```text
#!CommitTicketReference repository="sasmodels" revision="478882278efdd578c67d41a151c1209805308f3f"
added polydispersity back to fractal (as it used to be, as the
documentation implies and as core_shell_fractal already has). Also
formatted documentation according to current standard for references and
authorship/verification.  Fixes #963
```


* **butler** changed resolution from "" to "fixed"
* **butler** changed status from "assigned" to "closed"
